GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Content/textarea.config

```diff
@@ -256,11 +503,9 @@
   ]
 }]]></Value>
     </blocks>
-    <headingScaleStart>
-      <Value><![CDATA[]]></Value>
-    </headingScaleStart>
     <nextPage>
-      <Value><![CDATA[umb://document/7ebf39a4dbdd454b92e46e544af2a411]]></Value>
+      <Value Culture="cy"><![CDATA[]]></Value>
+      <Value Culture="en-gb"><![CDATA[umb://document/7ebf39a4dbdd454b92e46e544af2a411]]></Value>
     </nextPage>
   </Properties>
 </Content>
```

Comment:
When I run uSync > Everything > Import I get an error on this page:

![image](https://github.com/user-attachments/assets/07b7a1c6-f37a-4236-896d-4d4d6c472138)